src/llmcompressor/pipelines/layer_sequential/pipeline.py

```diff
@@ -51,7 +51,7 @@ def run_pipeline(
     # find layers
     layers = match_modules(model, sequential_targets)
 
-    with calibration_forward_context(model):
+    with calibration_forward_context(model), DisableQuantization(model):
```

Comment:
do we need to worry about the multi-context python 3.9 issue here? i think those changes are being reverted?

Reply:
No, see the comments I left in the PR's and issues around that. That was incorrect information that has been propagated around. Multi context statements have been enabled since Python 3.1. The piece that was added recently was enabling parentheses to group them.

Reply:
This is acceptable, since it's on a single line, it will work with both 3.9 and above; only thing we need to take care of going forward is to not use features introduced from 3.10+ onwards